Networking/Networking/ApplicationPassword/ApplicationPasswordUseCase.swift

```diff
@@ -28,3 +34,219 @@ protocol ApplicationPasswordUseCase {
     ///
     func deletePassword() async throws
 }
+
+final class DefaultApplicationPasswordUseCase: ApplicationPasswordUseCase {
+    /// WordPress.com Credentials.
+    ///
+    private let credentials: Credentials
+
+    /// SiteID needed when using WPCOM credentials
+    ///
+    private let siteID: Int64
+
+    /// To generate and delete application password
+    ///
+    private let network: Network
+
+    /// Stores the application password
+    ///
+    private let keychain: Keychain
+
+    /// Used to name the password in wpadmin.
+    ///
+    private var applicationPasswordName: String {
+        get async {
+            await UIDevice.current.model
+        }
+    }
+
+    init(siteID: Int64,
+         networkcredentials: Credentials,
+         network: Network? = nil,
+         keychain: Keychain = Keychain(service: "com.automattic.woocommerce.applicationpassword")) {
```

Comment:
We should use the same service name for Keychain in the whole app: `WooConstants.keychainServiceName`.

Reply:
Done in 6b63664ddea50ffe10b0e98463bba6ea3a67aa3e